packages/angular/src/schematics/application/schema.json

```diff
@@ -115,7 +115,7 @@
       "description": "The tool to use for running lint checks.",
       "type": "string",
       "enum": ["tslint", "eslint"],
-      "default": "tslint"
+      "default": "eslint"
```

Comment:
Might be worth holding off on this as the default until we have the actual ruleset for Angular? (Will be a follow up PR of mine this week). Right now you just given an ESLint config with nothing specific to Angular, and template HTML is not linted at all...

Therefore I would say _currently_, the TSLint experience is more appropriate for users expecting a complete Angular setup

Reply:
We actually set a default in angular.json, so this default will only apply if you choose to remove that one. But I'll revert it for now.